apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -63,6 +64,7 @@ export class AuthorizationService extends BasePermissionManager {
 		entityId: EntityId,
 		context: IPermissionContext
 	): Promise<boolean> {
+		// TODO: I think this try-catch is unnecessary and wrong because there can be different reasons why the entity cannot be loaded and they should bubble up.
```

Comment:
maybe create a ticket and remove this comment?

Reply:
I created a ticket and linked it in the comment, so future readers are aware of it. I think that's better than removing the comment. Or what do you think?